python/lsst/ts/externalscripts/make_love_stress_tests.py

```diff
@@ -0,0 +1,318 @@
+            title: StressLOVE v1
+            description: Configuration for StressLOVE
+            type: object
+            properties:
+              location:
+                description: Domain of the running LOVE instance to stress
+                type: string
+              number_of_clients:
+                description: The number of clients to create
+                type: number
+              number_of_messages:
+                description: The number of messages to store before calculating the mean latency
+                type: number
+              data:
+                description: List of CSC_name[:index]
+                type: array
+                minItems: 1
+                items:
+                    type: string
+            required: [location, number_of_clients,data]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def set_metadata(self, metadata):
+        """Compute estimated duration.
+
+        Parameters
+        ----------
+        metadata : SAPY_Script.Script_logevent_metadataC
+        """
+        # a crude estimate;
+        metadata.duration = len(self.clients) * 5
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Specify the Stress test configurations:
+        - LOVE host location
+        - Number of clients
+        - Number of messages to store
+        - CSCs
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Configuration with several attributes:
+
+            * location : a string, with the location of the
+                LOVE instance to stress
+            * number_of_clients : a number that indicates the number
+                of clients to create
+            * number_of_messages : a number that indicates the number
+                of messages to store before calculating the mean latency
+            * data : a list, where each element is a string in the form:
+
+                * CSC name and optional index as ``csc_name:index`` (a `str`).
+
+        Notes
+        -----
+        Saves the results on several attributes:
+
+        * location : a string, with the location of the LOVE instance to stress
+        * number_of_clients : a number, with the number of clients to create
+        * number_of_messages : a number, with the number of messages to store
+        * remotes : a dict of (csc_name, index): remote,
+            an `lsst.ts.salobj.Remote`
+
+        Constructing a `salobj.Remote` is slow (DM-17904), so configuration
+        may take a 10s or 100s of seconds per CSC.
+        """
+        self.log.info("Configure started")
+
+        # set configurations
+        self.location = config.location
+        self.number_of_clients = config.number_of_clients
+        self.number_of_messages = config.number_of_messages
+
+        # get credentials
+        self.username = "user"
```

Comment:
Any chance this can come from os.environ?

Reply:
I think at some point we can define the LOVE default users as environment variables. But at this moment we should have to add those variables for the `admin`, `user`, `cmd_user`, `authlist_user` and `test`. I suggest to leave this fixed for the moment.

Reply:
The password is already coming from an environment variable. I think it would probably be wise to use the same technique for the username. This way we don't have to update the script in the future..